wiki/Tournaments/CWC/2021/zh.md

```diff
@@ -2,6 +2,7 @@
 tags:
   - CWC
   - CWC2021
+outdated: true
```

Comment:
why is this outdated

Reply:
Translation hasn't been updated since the quarterfinals stage, could add an `outdated_since` for more context:
```suggestion
outdated: true
outdated_since: 909111c95fe5a038b47049d84e14d9ea068c09f5
```

If that isn't desirable, it can just be removed
```suggestion
```